src/system-fetch.js

```diff
@@ -66,9 +66,13 @@
       else
         url = url.substr(7);
       return fs.readFile(url, function(err, data) {
-        if (err)
-          return reject(err);
-        else {
+        if (err) {
+          if (err.errno == 34) {
```

Comment:
The error for file not found looks like this for me in Node - 
```
[Error: ENOENT, open '/Users/guybedford/Projects/es6-module-loader/asdf'
	Error loading file:///Users/guybedford/Projects/es6-module-loader/asdf]
```

Any ideas why it looks like an array here?

Also if I removed the _errno_ check here, or have another error like an access error, the error looks exactly the same and is no different - so what is this code doing then? Or is it only for certain versions of Node?